public/web_worker/patchBuildVariantsReduce.js

```diff
@@ -0,0 +1,37 @@
+// Iterate through PatchBuildVariants and determine if a task should be
```

Comment:
While i like this approach i think its important that we minimize code retyping.  And make it so pieces of code like this are more testable. Here are a few solutions i found to place this code within the src/ folder. 
We discussed this on slack. 

https://github.com/petersobolev/cra-worker/tree/master/src

https://github.com/educartoons/creact-react-app-typescript-web-workers

Reply:
I made [this ticket](https://jira.mongodb.org/browse/EVG-13522) to move webworkers into the /src directory when our build system allows.